code/modules/hydroponics/grown/melon.dm

```diff
@@ -76,3 +76,34 @@
 	to_chat(user, "<span class='warning'>[src] rapidly turns into ash!</span>")
 	qdel(src)
 	new /obj/effect/decal/cleanable/ash(drop_location())
+
+//Superior Melon
+/obj/item/seeds/superwatermelon
+	name = "pack of Superior Watermelon seeds"
+	desc = "These seeds grow into Superior Watermelon plants."
+	icon_state = "seed-superwatermelon"
+	species = "superwatermelon"
+	plantname = "Superior Watermelon Vines"
+	product = /obj/item/reagent_containers/food/snacks/grown/superwatermelon
+	rarity = 10
+	lifespan = 20
+	endurance = 20
+	instability = 80
+	growing_icon = 'icons/obj/hydroponics/growing_fruits.dmi'
+	icon_dead = "superwatermelon-dead"
+	genes = list(/datum/plant_gene/trait/repeated_harvest)
+	mutatelist = list(/obj/item/seeds/watermelon/holy)
```

Comment:
is this intended

Reply:
It is, the progression should be melon -> Super -> Holy

Reply:
but its melon -> holy or super

Reply:
ill edit the melon to mutate ONLY to super